config/defaultserverlist.json

```diff
@@ -9,6 +9,7 @@
     "Test Server GTNH 4 EU": "zeta.gtnewhorizons.com",
     "Prospercraft GTNH Server US": "newhorizonsold.prospercraft.com",
     "StoneLegion.com GTNH (Canada)": "mc.stonelegion.com",
+	"MyFTB GTNH German Server": "newhorizons.myftb.de",
```

Comment:
Just to clarify: this domain is different to [the one we remove](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/16821/files#diff-d09fbafc9d969165d4e2718faf79e02b17c6bd63db392f30b749a072295be28dL9). Is that correct?

Reply:
gtnewhorizons.myftb.de is the right one

Reply:
yes, just as it was before the removal. It's also on the website if  you open up the "Serveradressen" part https://myftb.de/servers/gt-new-horizons

Reply:
Ok will correct later